install/prepare_configuration_payload.md

```diff
@@ -38,48 +39,55 @@ For more description of these settings and the default values, see
-   * Set `ntp-pools` to reachable NTP pools.
-   * The `application_node_config.yaml` file is required. It is used to describe the mapping between prefixes in `hmn_connections.csv` and HSM subroles. This file also defines aliases application nodes. For details, see [Create Application Node YAML](create_application_node_config_yaml.md).
-   * For systems that use non-sequential cabinet id numbers, use `cabinets-yaml` to include the `cabinets.yaml` file. This file can include information about the starting ID for each cabinet type and number of cabinets which have separate command line options, but is a way to specify explicitly the id of every cabinet in the system. See [Create Cabinets YAML](create_cabinets_yaml.md).
+| `--bootstrap-ncn-bmc-user` `root` | Administrative account for the management node BMCs |
+| `--bootstrap-ncn-bmc-pass` `changeme` | Password for `bootstrap-ncn-bmc-user` account |
+| `--system-name` `eniac`  | Name of the HPE Cray EX system |
+| `--mountain-cabinets` `4` | Number of Mountain cabinets, but this could also be in `cabinets.yaml` |
+| `--starting-mountain-cabinet` `1000` | Starting Mountain cabinet ID |
+| `--hill-cabinets` `0` | Number of Hill cabinets, but this could also be in `cabinets.yaml` |
+| `--river-cabinets` `1` | Number of River cabinets, but this could also be in `cabinets.yaml` |
+| `--can-cidr` `10.103.11.0/24` | IP subnet for the CAN assigned to this system |
+| `--can-external-dns` `10.103.11.113` | IP address on CAN for this system's DNS server |
+| `--can-gateway` `10.103.11.1` | Virtual IP address for the CAN (on the spine switches) |
+| `--can-static-pool` `10.103.11.112/28` | MetalLB static pool on CAN |
+| `--can-dynamic-pool` `10.103.11.128/25` | MetalLB dynamic pool on CAN |
+| `--hmn-cidr` `10.254.0.0/17` | Override the default cabinet IPv4 subnet for River HMN |
+| `--nmn-cidr` `10.252.0.0/17` | Override the default cabinet IPv4 subnet for River NMN |
+| `--hmn-mtn-cidr` `10.104.0.0/17` | Override the default cabinet IPv4 subnet for Mountain HMN |
+| `--nmn-mtn-cidr` `10.100.0.0/17` | Override the default cabinet IPv4 subnet for Mountain NMN |
+| `--ntp-pools` `time.nist.gov` | External NTP pool(s) for this system to use |
+| `--site-domain` `dev.cray.com` | Domain name for this system |
+| `--site-ip` `172.30.53.79/20` | IP address and netmask for the PIT node `lan0` connection |
+| `--site-gw` `172.30.48.1` | Gateway for the PIT node to use |
+| `--site-nic` `p1p2` | NIC on the PIT node to become `lan0` |
+| `--site-dns` `172.30.84.40` | Site DNS servers to be used by the PIT node |
+| `--install-ncn-bond-members` `p1p1,p10p1` | NICs on each management node to become `bond0` |
+| `--application-node-config-yaml` `application_node_config.yaml` | Name of `application_node_config.yaml` |
+| `--cabinets-yaml` `cabinets.yaml` | Name of `cabinets.yaml` |
+| `--bgp-peers` aggregation | Override the default BGP peers, using aggregation switches instead of spines |
```

Comment:
If the literal argument to use is "aggregation", it should be enclosed in backticks.

```suggestion
| `--bgp-peers` `aggregation` | Override the default BGP peers, using aggregation switches instead of spines |
```